rules/Php56/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector.php

```diff
@@ -84,6 +85,14 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
+        $hasInnerHTML = (bool) $this->betterNodeFinder->findFirst(
+            (array) $node->stmts,
+            fn (Node $node): bool => $node instanceof InlineHTML
+        );
+        if ($hasInnerHTML) {
+            return null;
+        }
+
```

Comment:
This should be extracted to some HTML places. It's re-used in other rules too

Reply:
I moved to `InlineHTMLAnalyzer` service https://github.com/rectorphp/rector-src/pull/757/commits/f3cb07d61c271e8a230ce65376e2c633e2eeb63d